### ISSUE TITLE
Stable version for Nette 2.4

### ISSUE THREAD

**Issue by attreid** [Kdyby/FormsReplicator#46]
will you release stable version for nette/forms 2.4?

**Comment by jspetrak** [comment]
Is that a matter of version update in `composer.json`, or more work is needed?

**Comment by fprochazka** [comment]
Sure, once the compatibility is fixed :) Sadly, it will probably require a not-so-small refactoring.

**Comment by fprochazka** [comment]
Btw, have you seen the big banner in readme? https://github.com/Kdyby/FormsReplicator#save-me-please

**Comment by solcik** [comment]
I just released the version [v2.0.0-rc1](https://github.com/Kdyby/FormsReplicator/releases/tag/v2.0.0-rc1) for php 7.1 and nette v3.0.

I will be happy to merge a PR (if you prepare one) for a stable release for nette v2.4.

**Comment by solcik** [comment]
Please test the release for Nette ```v2.4``` via branch [nette-2.4](https://github.com/Kdyby/FormsReplicator/tree/nette-2.4)

Tests are green. I am going to release it in about a week as ```v1.4.0```

## Kdyby/FormsReplicator
```
	"require": {
		"php": ">=5.6,<7.2",
		"nette/forms": "2.4.*"
	},
```

## Yours

```
	"require": {
		"php": ">=5.6,<7.2",
		"nette/forms": "^2.4.9",
		"kdyby/forms-replicator": "dev-nette-2.4"
	},
```

**Comment by radekdostal** [comment]
Why PHP < 7.2? Nette 2.4 is compatible up to PHP 7.3.

**Comment by solcik** [comment]
I'm going to double check. In which nette version was removed Object class?

**Comment by radekdostal** [comment]
I don't know, but I use FormsReplicator with Nette 2.4 (PHP 7.2, 7.3) up to commit 10133eb2f2bf7aa445ab4a0a7e8701c1f5ffeded.

**Comment by solcik** [comment]
I reflected real dependencies in 3de1034 and added php 7.2  & 7.3 to Travis (f5627e5ee14e).

Everything seems to be fine.

**Comment by radekdostal** [comment]
Great. It works.

**Comment by solcik** [comment]
Released as [v1.4.0](https://github.com/Kdyby/FormsReplicator/releases/tag/v1.4.0).

**Comment by solcik** [comment]
There is also a release [v2.0.0](https://github.com/Kdyby/FormsReplicator/releases/tag/v2.0.0) for php ```^7.1``` & Nette ```v3```.
